### PR TITLE
1.0.11 - Use WMI event for airplane mode on serw12

### PR DESCRIPTION
**Pull request by jackpot51**
This should prevent EC hangs under load with keys pressed continuously

Rebase and tag when approved

### CODE REVIEW

**Comment by jackpot51** [comment]
Looking at the ACPI tables, it looks like HID event filter is not used. Airplane mode instead uses a WMI event that doesn't require running the poll daemon. I will adjust this PR when I have time

File: system76_input.c [hunk 1]
```diff
@@ -113,16 +107,17 @@ static void s76_input_screen_wmi(void) {
...
-  }
 
-	if (unlikely(IS_ERR(s76_input_polling_task))) {
-		s76_input_polling_task = NULL;
-		S76_ERROR("Could not create polling thread\n");
-		return PTR_ERR(s76_input_polling_task);
+		if (unlikely(IS_ERR(s76_input_polling_task))) {
+			s76_input_polling_task = NULL;
+			S76_ERROR("Could not create polling thread\n");
+			return PTR_ERR(s76_input_polling_task);
```
Comment (crawfxrd) [inline system76_input.c:119]: Worth fixing this? Value is set to NULL before getting the error.

Comment (jackpot51) [reply system76_input.c]: Fixed in latest commit

**Review by crawfxrd — APPROVED** [review]

**Review by jacobgkau — APPROVED** [review]
Airplane mode key works on serw12 and polling thread doesn't run. Airplane mode key still works on other devices that use `DRIVER_AP_KEY` and the polling thread (tested with bonw13.)

**Pull request merged**